fontforgeexe/cvexportdlg.c

```diff
@@ -55,6 +56,149 @@ struct sizebits {
 #define CID_Size	1000
 #define CID_Bits	1001
 
+static int EXPP_OK(GGadget *g, GEvent *e) {
+    if ( e->type==et_controlevent && e->u.control.subtype == et_buttonactivate )
+	*(int *) GDrawGetUserData(GGadgetGetWindow(g)) = true;
+return( true );
+}
+
+static int expp_e_h(GWindow gw, GEvent *event) {
+    if ( event->type==et_close ) {
+	*(int *) GDrawGetUserData(gw) = true;
+    } else if ( event->type == et_char ) {
+	/*
+	if ( event->u.chr.keysym == GK_F1 || event->u.chr.keysym == GK_Help ) {
+	    help("XXX");
+	    return( true );
+	}
+	*/
+	return( false );
+    }
+    return( true );
+}
+
+void _ExportParamsDlg(ExportParams *ep) {
+    GRect pos;
+    GWindow gw;
+    GWindowAttrs wattrs;
+    GGadgetCreateData gcd[5], boxes[4], *hvarray[5][4], *barray[4];
+    GTextInfo label[5];
+    int done = false;
```

Comment:
Why not make this `bool`

Reply:
This code is copied from the import dialog case and I didn't want to change the `GetUserData` casts.